src/client/app/_shared/item.component.ts

```diff
@@ -8,7 +8,7 @@ import { Player } from '../../../models/player';
 import { includes } from 'lodash';
 import { ColyseusGameService } from '../colyseus.game.service';
 
-export type MenuContext = 'Sack' | 'Belt' | 'Ground' | 'GroundGroup' | 'Equipment' | 'Left' | 'Right' | 'Coin';
+export type MenuContext = 'Locker' | 'Sack' | 'Belt' | 'Ground' | 'GroundGroup' | 'Equipment' | 'Left' | 'Right' | 'Coin';
```

Comment:
Call it a Kubby instead. You can't use Locker as the context because L is Left Hand already.